services/server/server.go

```diff
@@ -331,6 +335,21 @@ func (s *Server) UpdateStatus(ctx context.Context, req *serverpb.UpdateStatusReq
 	}, nil
 }
 
+// StartSecurityChecks starts STT checks execution.
+func (s *Server) StartSecurityChecks(ctx context.Context, request *serverpb.StartSecurityChecksRequest) (*serverpb.StartSecurityChecksResponse, error) {
+
```

Comment:
we usually don't leave blank lines as first line in method.

Reply:
Fixed